components/ListHeader/list-header.tsx

```diff
@@ -44,7 +44,7 @@ const ListHeader = ({ name, isPublic, listId, isOwner, numberOfContributors }: L
         </div>
         <div className="flex flex-col justify-center p-2 header-info grow">
           <div className="flex gap-2">
-            <Title level={1} className="!text-2xl font-semibold tracking-tight text-slate-900">
```

Comment:
`tracking-tight` is removed here and in the other files below, because the base style handles it now.